### PR TITLE
new feature: audit manifest file from command line args

### PR DESCRIPTION
**Pull request by jinankjain**
I started working on the new feature to audit config files passed as a command line argument. Right now it works only for securityContext. It needs to be ported to other features too.

I am not happy about of lot of stuff in this PR mostly regarding code quality. Need lot of interface love.  Sorry I was in hurry could not make code more clear.

It is just a proof of concept that is possible to do so.

### CODE REVIEW

File: cmd/root.go [hunk 1]
```diff
@@ -15,6 +15,7 @@ type rootFlags struct {
 	verbose    bool
 	allPods    bool
 	json       bool
+	config     string
```
Comment (JonPulsifer) [inline cmd/root.go]: I think it should be named something other than config, given that the names of the YAML files we anticipate to audit with this are called manifests.

File: cmd/root.go [hunk 2]
```diff
@@ -41,4 +42,5 @@ func init() {
 	RootCmd.PersistentFlags().BoolVarP(&rootConfig.verbose, "verbose", "v", false, "Enable debug (verbose) logging")
 	RootCmd.PersistentFlags().BoolVarP(&rootConfig.json, "json", "j", false, "Enable json logging")
 	RootCmd.PersistentFlags().BoolVarP(&rootConfig.allPods, "allPods", "a", false, "Audit againsts pods in all the phases (default Running Phase)")
+	RootCmd.PersistentFlags().StringVarP(&rootConfig.config, "config", "y", "", "yaml configuration to audit")
```
Comment (JonPulsifer) [inline cmd/root.go]: I think the flag should be `-f`, staying in line w/ `kubectl` and other tools

**Comment by klautcomputing** [comment]
@jinankjain Want to take a look and tell me what you think about my changes?

**Comment by jinankjain** [comment]
Everything works now could be squashed into one commit and mergerd 👍  @klautcomputing

**Pull request merged**